docs/user/getting_started.rst

```diff
@@ -237,6 +237,11 @@ GLUON_RELEASE
   is available. The same GLUON_RELEASE has to be passed to ``make`` and ``make manifest``
   to generate a correct manifest.
 
+GLUON_SITE_VERSION
+  Version of the site configuration. This string is displayed in the config mode
+  and `gluon-info`. If not set, Gluon generates a version-string using `git describe`
+  of the site folder.
```

Comment:
```suggestion
  in the site folder.
```

or something like that. of sounds wrong to me

Reply:
should be `on`